crates/starknet-types-core/src/felt/papyrus_encoding.rs

```diff
@@ -0,0 +1,71 @@
+use std::io::Error;
+
+use super::Felt;
+/// Storage efficient serialization for field elements.
+impl Felt {
+    // Felt encoding constants.
+    const CHOOSER_FULL: u8 = 15;
+    const CHOOSER_HALF: u8 = 14;
+    pub fn serialize(&self, res: &mut impl std::io::Write) -> Result<(), Error> {
```

Comment:
if we are talking about serialize, maybe call the file and the feature "papyrus_serialization"